src/components/FilterPanel.tsx

```diff
@@ -40,32 +40,19 @@ export default function FilterPanel({
   const [dialogOpen, setDialogOpen] = useState(false);
   const [params, setQueryParams] = useQueryParams();
 
-  const [minAmount, setMinAmount] = useState(params.minAmount);
-  const [grades, setGrades] = useState(params.grades);
-  const [majors, setMajors] = useState(params.majors);
-  const [states, setStates] = useState(params.states);
-  const location = useLocation();
-
-  useEffect(() => {
-    const query = queryString.parse(location.search, {
-      arrayFormat: 'bracket-separator',
-      arrayFormatSeparator: ',',
-      parseNumbers: true,
-    });
-
-    setMinAmount(query.minAmount);
-    setGrades(query.grades);
-    setMajors(query.majors);
-    setStates(query.states);
-  }, [location]);
```

Comment:
I dont remember why we had this but I dont think we need it either way.

Reply:
The effect runs when the location gets changed, e.g. let's say the user clicks on one of the chips from the FilterBar. What I don't remember is why `useQueryParams` doesn't take care of this itself.

Reply:
I found the [comment](https://github.com/beyondhb1079/s4us/pull/969#discussion_r839102427) where I commented on this but I guess I never asked why this was needed in the first place 🤔 . Yeah I don't think this is necessary as `useQueryParams` uses `useLocation` which should cause a re-render and a re-initialization of state.